File: FaceSample/app/src/main/java/com/regula/facesample/items/customization/LivenessProcessingCustomItem.java
package com.regula.facesample.items.customization;

import android.content.Context;

import com.regula.facesample.items.CategoryItem;
import com.regula.facesample.items.customization.fragment.LivenessProcessingCustomFragment;
import com.regula.facesample.util.LivenessResponseUtil;
import com.regula.facesdk.FaceSDK;
import com.regula.facesdk.configuration.LivenessConfiguration;

/**
 * Created by Sergey Yakimchik on 29.04.22.
 * Copyright (c) 2022 Regula. All rights reserved.
 */

public class LivenessProcessingCustomItem extends CategoryItem {

   @Override
   public void onItemSelected(Context context) {
      LivenessConfiguration configuration = new LivenessConfiguration.Builder()
              .registerProcessingFragment(LivenessProcessingCustomFragment.class)
              .build();
      FaceSDK.Instance().startLiveness(context, configuration, livenessResponse -> LivenessResponseUtil.response(context, livenessResponse));
   }

   @Override
   public String getDescription() {
      return "Customize liveness processing and retry screens";
   }

   @Override
   public String getTitle() {
      return "Custom liveness process";
   }
}

File: FaceSampleKotlin/app/src/main/java/com/regula/facesamplekotlin/items/customization/LivenessProcessingCustomItem.kt
package com.regula.facesamplekotlin.items.customization

import android.content.Context
import com.regula.facesamplekotlin.category.CategoryItem
import com.regula.facesamplekotlin.items.fragments.LivenessProcessingCustomFragment
import com.regula.facesamplekotlin.util.LivenessResponseUtil
import com.regula.facesdk.FaceSDK
import com.regula.facesdk.configuration.LivenessConfiguration
import com.regula.facesdk.model.results.LivenessResponse

class LivenessProcessingCustomItem : CategoryItem() {

    override fun onItemSelected(context: Context) {
        val configuration = LivenessConfiguration.Builder()
            .registerProcessingFragment(LivenessProcessingCustomFragment::class.java)
            .build()
        FaceSDK.Instance().startLiveness(
            context, configuration
        ) { livenessResponse: LivenessResponse -> LivenessResponseUtil.response(context, livenessResponse) }
    }

    override val title: String
        get() = "Custom liveness process"

    override val description: String
        get() = "Customize liveness processing and retry screens"
}